app/controllers/prison/visits_controller.rb

```diff
@@ -72,4 +80,30 @@ def visit_page(visit)
       prison_deprecated_visit_path(visit)
     end
   end
+
+  def load_visit
```

Comment:
Also all these methods are now in a booking response context:

https://github.com/ministryofjustice/prison-visits-2/blob/e8552b47a2e1e04a4c58e5bdd62947c8f6979521/app/controllers/concerns/booking_response_context.rb